### PR TITLE
Fresh pot

### PR DESCRIPTION
**Pull request by bmjen**

### CODE REVIEW

File: locales/pdk.pot [hunk 1]
```diff
@@ -806,64 +842,80 @@ msgstr ""
 msgid "Invalid JSON in metadata.json: %{msg}"
 msgstr ""
 
-#: ../lib/pdk/module/metadata.rb:114
+#: ../lib/pdk/module/metadata.rb:92
```
Comment (jbondpdx) [inline locales/pdk.pot:845]: Could we add periods at the ends of these new strings, please?

File: locales/pdk.pot [hunk 2]
```diff
@@ -972,88 +1024,128 @@ msgstr ""
...
-#: ../lib/pdk/util/git.rb:47
+#: ../lib/pdk/util/bundler.rb:212
+msgid "Unable to install requested binstubs."
+msgstr ""
+
+#: ../lib/pdk/util/git.rb:64
 msgid "Unable to access the template repository \"%{repository}\""
 msgstr ""
 
+#: ../lib/pdk/util/puppet_version.rb:22
```
Comment (jbondpdx) [inline locales/pdk.pot:1121]: these need periods at the end, too

**Review by jbondpdx — CHANGES_REQUESTED** [review]
Looks good, thank you! Sorry to be a stickler about the periods, but consistency.

File: locales/pdk.pot [hunk 3]
```diff
@@ -806,64 +842,80 @@ msgstr ""
...
+#: ../lib/pdk/module/metadata.rb:92
+msgid "Module metadata does not contain any requirements"
+msgstr ""
+
+#: ../lib/pdk/module/metadata.rb:93
+msgid "Module metadata does not contain a \"puppet\" requirement"
+msgstr ""
+
+#: ../lib/pdk/module/metadata.rb:94
+msgid "\"puppet\" requirement in module metadata does not specify a \"version_requirement\""
```
Comment (bmjen) [inline locales/pdk.pot:854]: @jbondpdx should this sentence start with "The", so that we aren't starting with a "?

Comment (jbondpdx) [reply locales/pdk.pot:854]: Yeah, I think that's a good idea.

**Pull request closed without merging**